### DISCUSSION TITLE
essay/2024/05/19/blog-platform-evolution

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#211] [Comments]
# essay/2024/05/19/blog-platform-evolution

블로그 플랫폼을 세 번 변경했습니다.

https://jeho.page/essay/2024/05/19/blog-platform-evolution.html

**Comment by asomu** [comment]
검색때문에 그럴까요? 지금도 네이버는 개발자 블로그로 별로라는 생각이 듭니다.

**Reply by BenjaminKim** [reply]
검색 노출, 이웃과의 소통, 기능과 디자인의 꾸준한 개선이 좋다고 생각합니다.
